front/lib/iam/errors.ts

```diff
@@ -9,3 +9,15 @@ export class AuthFlowError extends Error {
     super(message);
   }
 }
+
+export class UnverifiedEmailError extends Error {
```

Comment:
We should avoid creating too many variation of errors if they don't contain more logic. What about using `AuthFlowError` for our flow?